daemon/graphdriver/driver.go

```diff
@@ -46,7 +48,6 @@ type ProtoDriver interface {
 	// String returns a string representation of this driver.
 	String() string
 	// Create creates a new, empty, filesystem layer with the
-	// specified id and parent. Parent may be "".
```

Comment:
Ugh--thanks for catching this @2opremio and thanks for quickly correcting it @HuKeping :)